content/execution_model.tex

```diff
@@ -1,18 +1,18 @@
 An \openshmem program consists of a set of \openshmem processes called \acp{PE}
-that execute in a \ac{SPMD}-like model where each \ac{PE} can take a different
+that execute in a \ac{SPMD}-like model, where each \ac{PE} can take a different
```

Comment:
See _Comma Abuse No. 15_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).